internal/locate/region_request.go

```diff
@@ -1294,15 +1294,11 @@ func (s *replicaSelector) canFallback2Follower() bool {
 	if s == nil || s.state == nil {
 		return false
 	}
-	state, ok := s.state.(*accessFollower)
-	if !ok {
-		return false
-	}
-	if !state.isStaleRead {
+	_, ok := s.state.(*accessKnownLeader)
+	if ok {
 		return false
 	}
-	// can fallback to follower only when the leader is exhausted.
-	return state.lastIdx == state.leaderIdx && state.IsLeaderExhausted(s.replicas[state.leaderIdx])
+	return true
```

Comment:
This change may reduce a lot of `tikvServerBusy` backoff, and may need to be discussed. @cfzjywxk @you06 @zyguan PTAL